src/main/java/com/jalasoft/webservice/controller/FileManager.java

```diff
@@ -29,6 +29,7 @@ public static String getFilePath(@RequestParam("file") MultipartFile file) throw
         Utils utils = new Utils();
         String filePath = utils.getTemp() + file.getOriginalFilename();
         Path location = Paths.get(filePath);
+        System.out.println(filePath);
```

Comment:
System.out.println must not be in our code.

Reply:
Solved